### PR TITLE
fix some javadoc typos

### PR DESCRIPTION
**Pull request by wowselim**

### CODE REVIEW

File: knot/src/main/kotlin/de/halfbit/knot/CompositeKnot.kt [hunk 1]
```diff
@@ -11,14 +11,14 @@ import java.util.concurrent.atomic.AtomicBoolean
 import kotlin.reflect.KClass
 
 /**
- * If your [Knot] becomes big and you want to improve its redability and maintainability, you may consider
+ * If your [Knot] becomes big and you want to improve its readability and maintainability, you may consider
  * to decompose it. You start decomposition by grouping related functionality into, in a certain sense,
- * indecomposable pieces called `Primes`.
+ * undecomposable pieces called `Primes`.
```
Comment (beworker) [inline knot/src/main/kotlin/de/halfbit/knot/CompositeKnot.kt]: How about this? https://en.wikipedia.org/wiki/Indecomposable

Comment (wowselim) [reply knot/src/main/kotlin/de/halfbit/knot/CompositeKnot.kt]: looks like the `in` version is used a lot in mathematical contexts. it just felt odd when I read it :)

Comment (beworker) [reply knot/src/main/kotlin/de/halfbit/knot/CompositeKnot.kt]: Look here: https://en.wikipedia.org/wiki/Prime_knot ;-)

Comment (wowselim) [reply knot/src/main/kotlin/de/halfbit/knot/CompositeKnot.kt]: fair enough 😉
reverted this change

**Pull request merged**